platform-hub-api/config/secrets.yml

```diff
@@ -10,13 +10,17 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
+common: &common
+  secret_key_base: <%= ENV.fetch('SECRET_KEY_BASE') { raise 'SECRET_KEY_BASE missing from env' } %>
+  github_client_id: <%= ENV.fetch('GITHUB_CLIENT_ID') { raise 'GITHUB_CLIENT_ID missing from env' } %>
+  github_client_secret: <%= ENV.fetch('GITHUB_CLIENT_SECRET') { raise 'GITHUB_CLIENT_SECRET missing from env' } %>
+
+
 development:
-  secret_key_base: c6c7fb336cd9983d677318ccca486149658bb14eb9f88a0fddf90f936d4e5d328ffc5d1c06576f99b4df6659e45e216f1917abbc9c17a186e252fce424c2653e
+  <<: *common
 
 test:
-  secret_key_base: afd76d611a7ae59d56ead6888e6b6b0f51f31e1fbbf4e865eb3f19e0bbfde9e9e7c1f5e8d1dc2888888c7d41417a4310f221749384a46fe6b9bd672f491bccfa
```

Comment:
Not this key specifically, but the one provided via an env var. It's used to encrypt the ID of the user account during the OAuth flow with GitHub (in the `state` parameter), so we can know – in the callback – which user to associate that GitHub identity with.